### DISCUSSION TITLE
[PROPOSAL] Relax daily SFL limit before the halving

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2427] [General]
Please consider disabling the daily SFL limit in the week before the halving.  I haven't run into this UI myself (yet) but it's kind of the most terrifying and surprising UI in the game IMO.

![image](https://user-images.githubusercontent.com/36871683/231225959-19972c2f-2613-40da-9c57-2108bf93ca5f.png)

**Comment by future2back** [comment]
I will never face it! 🤣
